### PR TITLE
1810- Mary

### PR DESCRIPTION
**Pull request by mgoodhart5**
scary scrabble challenge

### CODE REVIEW

File: lib/scrabble.rb [hunk 1]
```diff
@@ -1,13 +1,37 @@
...
-      "A"=>1, "B"=>3, "C"=>3, "D"=>2,
-      "E"=>1, "F"=>4, "G"=>2, "H"=>4,
-      "I"=>1, "J"=>8, "K"=>5, "L"=>1,
-      "M"=>3, "N"=>1, "O"=>1, "P"=>3,
-      "Q"=>10, "R"=>1, "S"=>1, "T"=>1,
-      "U"=>1, "V"=>4, "W"=>4, "X"=>8,
-      "Y"=>4, "Z"=>10
-    }
+
+  def initialize
```
Comment (memcmahon) [inline lib/scrabble.rb:3]: You don't need to have an initialize - you could have just kept the point_values method and used its return value to access the hash!

File: lib/scrabble.rb [hunk 2]
```diff
@@ -1,13 +1,37 @@
...
+   "E"=>1, "F"=>4, "G"=>2, "H"=>4,
+   "I"=>1, "J"=>8, "K"=>5, "L"=>1,
+   "M"=>3, "N"=>1, "O"=>1, "P"=>3,
+   "Q"=>10, "R"=>1, "S"=>1, "T"=>1,
+   "U"=>1, "V"=>4, "W"=>4, "X"=>8,
+   "Y"=>4, "Z"=>10
+     }
+  end
+
+  def score(turn)
```
Comment (memcmahon) [inline lib/scrabble.rb:19]: Great - super clean!

File: test/scrabble_test.rb [hunk 3]
```diff
@@ -5,8 +5,17 @@
 
 class ScrabbleTest < Minitest::Test
```
Comment (memcmahon) [inline test/scrabble_test.rb:6]: I would also like to see a test for an empty string and nil being passed in as arguments.

File: lib/scrabble.rb [hunk 4]
```diff
@@ -1,13 +1,37 @@
...
+   answer = turn.upcase.chars.map do |letter|
+     letter = alphabet[letter]
+   end
+   answer.sum
+  end
+
+  def score_with_multipliers(turn, multipliers)
```
Comment (memcmahon) [inline lib/scrabble.rb:26]: Looks good!

**Review by memcmahon — COMMENTED** [review]
Great job!
